collect_app/src/main/res/layout/activity_geo.xml

```diff
@@ -31,19 +31,23 @@
             tools:text="@string/please_wait_long"
             tools:visibility="visible" />
 
-        <FrameLayout
-            android:id="@+id/map_container"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent" />
-
     </LinearLayout>
 
+    <FrameLayout
+        android:id="@+id/map_container"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@+id/linearlayout" />
+
     <ScrollView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_below="@+id/linearlayout"
         android:layout_gravity="end"
         android:paddingBottom="10dip"
-        android:paddingTop="50dip">
+        android:paddingTop="10dip">
```

Comment:
Attributes `layout_gravity`,  `padding_bottom` & `padding_top` are not required now